pipeline/src/base.py

```diff
@@ -32,6 +32,36 @@ def to_jsonld(self, include_empty_properties=True, embed_linked_nodes=True, with
         """
         docstring goes here
         """
+
+        def value_to_jsonld(value):
+            if isinstance(value, LinkedMetadata):
+                if embed_linked_nodes:
+                    item = value.to_jsonld(
+                        with_context=False,
+                        include_empty_properties=include_empty_properties,
+                        embed_linked_nodes=embed_linked_nodes
+                    )
+                else:
+                    if hasattr(value, "id") and value.id is None:
+                        raise ValueError("Exporting as a stand-alone JSON-LD document requires @id to be defined.")
+                    item = {
+                        "@id": value.id,
+                        "@type": value.type_
```

Comment:
did we clarify if the additional setting of `@type` is valid for JSON-LD?

Reply:
that's a separate issue, will be addressed in a future PR